spring-cloud-task-core/src/test/java/org/springframework/cloud/task/repository/dao/MapTaskExecutionDaoTests.java

```diff
@@ -16,6 +16,10 @@
 
 package org.springframework.cloud.task.repository.dao;
 
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
```

Comment:
Static imports should be a the end of the import list: https://github.com/spring-projects/spring-framework/wiki/Code-Style#import-statements